regression/contracts-dfcc/chain.sh

```diff
@@ -43,6 +43,10 @@ else
   $goto_cc -o "${name}${dfcc_suffix}.gb" "${name}.c"
 fi
 
+if [[ "${args_inst}" != *"malloc"* ]]; then
+  args_inst="--no-malloc-fail $args_inst"
```

Comment:
Please don't do that. It will be outrageously difficult to understand what tests actually do.

Reply:
Hi Daniel, we're handling this in a different PR (#8106) where we're reverting this change across all `regression/` build files.

It would be preferable if we can merge this one with this change in, so that it's on par with the other build changes *currently* in `develop`, so that we can then track and remove all of them in #8106 instead of having some of them here and some there, increasing the chance of having some stale changes by accident.